collections/landsat-8-l2.yaml

```diff
@@ -138,8 +132,6 @@ CubeDimensions:
       - B05
       - B06
       - B07
-      - B08
-      - B09
       - B10
       - B11
```

Comment:
In the [SH docs](https://docs.sentinel-hub.com/api/latest/data/landsat-8-l2/#available-bands-and-data) it looks like there is no B11 in L8 L2 either.

Reply:
Somehow missed that, removed.